index.html

```diff
@@ -0,0 +1,42 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <section>
+      <nav id="navbar">
+        <div id="logo">
+          <img src="images/image_geometry_1.svg" width="111" height="74">
+        </div>
+        <div id="icmenu">
+          <img src="images/ic_menu.svg" width="40" height="28">
+        </div>
+      </nav>
+    </section>
+    <section>
+      <div class="headline">
+        <div id="name">
+          Mehmet Selçuk
+          Güler
+        </div>
+        <div id="message">

```

Comment:
- [ ] [OPTIONAL] You should kindly prefer using `class` selectors instead of `id`. This is because  **a class can be used to identify more than one HTML element.** :ok_hand: